### PR TITLE
Add admin guide for glusterfs-coreutils

### PR DESCRIPTION
**Pull request by anoopcs9**
General overview on how to install and use glusterfs-coreutils package.

closes gluster/glusterfs-coreutils#9

Signed-off-by: Anoop C S <anoopcs@redhat.com>

### CODE REVIEW

File: Administrator Guide/GlusterFS Coreutils.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+As mentioned above glusterfs coreutils also provides standalone commands to perform the basic GNU coreutil functionalities. All those commands are prepended by 'gf'. Instead of invoking a gluster client-shell you can directly make use of these to establish and perform the operation in one shot. For example see the following sample usage of gfstat command:
+
+    gfstat glfs://localhost/vol/foo
+
+There is an exemption regarding flock coreutility which is not available as a standalone command for a reason described under 'Notes' section.
+
+For more information on each command and corresponding options see associated man pages.
+
+## Notes
+* Within a particular session of gluster client-shell, history of commands are preserved i.e, you can use up/down arrow keys to search through previously executed commands or the reverse history serach technique using Ctrl+R.
```
Comment (prashanthpai) [inline Administrator Guide/GlusterFS Coreutils.md]: s/serach/search

File: Administrator Guide/GlusterFS Coreutils.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    dnf install glusterfs-coreutils
+OR
+
+    yum install glusterfs-coreutils
+
+RPMs are also available at <http://download.gluster.org/gluster/glusterfs-coreutils/>
+
+## Usage
+glusterfs-coreutils provides a set of basic utilities such as cat, cp, flock, ls, mkdir, rm, stat and tail that are implemented specifically using the GlusterFS API commonly known as libgfapi. These utilities can be used either inside a gluster remote
+shell or as standalone commands with 'gf' prepended to their respective base names. For example, glusterfs cat utility is named as gfcat ans so on with an exception to flock core utility for which a standalone gfflock command is not provided as such(see the notes section on why flock is designed in that way).
```
Comment (prashanthpai) [inline Administrator Guide/GlusterFS Coreutils.md]: s/ans/and

**Comment by prashanthpai** [comment]
Pointed out few nitpicks. Will merge this after trying out the steps mentioned.

**Comment by anoopcs9** [comment]
Re-based PR after addressing initial comments.

**Comment by prashanthpai** [comment]
@anoopcs9 Please let me know once the package is available.

**Comment by prashanthpai** [comment]
This repo had it's git history re-written on 19 May 2016. Please delete your existing fork and create a fresh fork. You will have to re-send the PR against the latest repo.

**Pull request closed without merging**